bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/core/listeners/SetupListener.java

```diff
@@ -56,7 +56,7 @@ private boolean compatible(Alloc pred, Alloc succ) {
 		return 
 			pred != null &&
 			pred.getObs().getInstrumentString().equals(succ.getObs().getInstrumentString()) &&
-			pred.getObs().getCoords().equals(succ.getObs().getCoords());
+			pred.getObs().getCoords(pred.getMiddlePoint()).equals(succ.getMiddlePoint());
```

Comment:
Looking at this closely, it appears that we're comparing `WorldCoords` to a `Long` which is never going to be equals.

It's counterintuitive perhaps but maybe the time to use for both the `pred` and `succ` observation would be the `pred` interval end or the `succ` interval start.  The idea is that you can continue observing without a new setup if the two observations are in the same place, which that would presumably tell you.